plugins/cylance_protect/help.md

```diff
@@ -31,13 +31,13 @@ You must create a Custom Application by following this procedure from the Cylanc
   
 The connection configuration accepts the following parameters:  
 
-|Name|Type|Default|Required|Description|Enum|Example|
-| :--- | :--- | :--- | :--- | :--- | :--- | :--- |
-|application_id|credential_secret_key|None|True|Application ID for CylancePROTECT instance|None|1abc234d-5efa-6789-bcde-0f1abcde23f5|
-|application_secret|credential_secret_key|None|True|Generated token that allows access to Cylance Resources|None|1abc234d-5efa-6789-bcde-0f1abcde23f5|
-|tenant_id|credential_secret_key|None|True|The unique tenant ID of the tenant that the device belongs to|None|1abc234d-5efa-6789-bcde-0f1abcde23f5|
-|url|string|https://protectapi.cylance.com|True|Web API URL|None|https://protectapi.cylance.com|
-  
+|Name|Type|Default|Required|Description|Enum|Example|Placeholder|Tooltip|
```

Comment:
Is this a plugin tooling bug? Might be worth manually removing these additions and making a ticket

Reply:
[All of the prior snyk tickets (which have been merged) have this in the help.md](https://github.com/rapid7/insightconnect-plugins/pull/2910/files#diff-c9045ce4af3162c1cb15f5e8f36b945bcef925c9ac433c217b00f9940e419ca7L41). I am assuming as these plugins (that have snyk vulns) are relatively old, it's more likely refactoring to match the modern help.md. The example input and output fields remain the same so it should be ok.